src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1289,13 +1289,26 @@ private DetectProperties() {
         new DetectProperty<>(new EnumProperty<>("logging.level.com.synopsys.integration", LogLevel.INFO, LogLevel.class))
             .setInfo("Logging Level", DetectPropertyFromVersion.VERSION_5_3_0)
             .setHelp("The logging level of Detect.",
-                "Detect logging is performed using Spring Boot's default logging setup (Logback). Detect sets the default log message format to \"%d{yyyy-MM-dd HH:mm:ss z} ${LOG_LEVEL_PATTERN:%-6p}[%thread] %clr(---){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}\". You can change your log message format by setting the Spring Boot <i>logging.pattern.console</i> property to a different pattern. Refer to the Spring Boot logging documentation for more details.")
+                "INFO level logging is recommended for normal use to keep the size of the log reasonable. Troubleshooting requires DEBUG or TRACE.<p/>" +
+                    "Detect logging is performed using Spring Boot's default logging setup: Logback. " +
+                    "The general format of this property name is <i>logging.level.{package}[.{class}]</i>. " +
+                    "The property name shown above specifies package <i>com.synopsys.integration</i> because that is the name of Detect's top-level package. " +
+                    "Changing the logging level for that package changes the logging level for all Detect code, as well as Synopsys integration libraries that Detect uses. " +
+                    "Changing the logging level for package <i>com.synopsys.integration</i> does not change the logging level for non-Synopsys libraries that Detect uses because their base package names are different. " +
```

Comment:
Change to read: " Non-Synopsys libraries that Detect uses are not affected."

Reply:
Done